vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -965,44 +987,23 @@ func TestDurangoDisabledTransactions(t *testing.T) {
 		{
 			name: "AddValidatorTx",
 			buildTx: func(env *environment) *txs.Tx {
-				ins, unstakedOuts, stakedOuts, signers, err := env.utxosHandler.Spend(
```

Comment:
This is what I mean by building a tx "manually". The tx is supposed to pass verification, so we really should use the txBuilder to do that. This also shorten the test